src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -87,8 +87,12 @@ internal void Clear()
 			var adjustedLineWidth = borderThickness.Top;
 			var adjustedLineWidthOffset = adjustedLineWidth / 2;
 
-			var adjustedArea = area;
-			adjustedArea.Inflate(-adjustedLineWidthOffset, -adjustedLineWidthOffset);
+			var adjustedArea = new Rect(
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'adjustedArea'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)